### ISSUE TITLE
Just a Question to Community

### ISSUE THREAD

**Issue by mgip** [AsgardCms/Platform#826]
The concept of this project is very interenting.

But i would to know if this project is still supported? 

Thanks Guys

**Comment by arturmamedov** [comment]
I think not more

**Comment by abdallhsamy** [comment]
the creator not, but we still use it , still support it and still love it

**Comment by arturmamedov** [comment]
Yeah, i use it for my own CMS with also a multi tenant integration, and it works like a charm!

**Comment by dineshsailor** [comment]
Using from long time for my multiple project, as well as keep upgrading with latest library & package

**Comment by arturmamedov** [comment]
@dineshsailor Great! I am not upgraded, still use laravel 5

**Comment by abdallhsamy** [comment]
> @dineshsailor Great! I am not upgraded, still use laravel 5

we are going to use Laravel 10, you're great 😃
